source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,571 @@ Filters Builders
    :class: singlecol
 
 .. _filters-builders:
+
+Overview 
+--------
+In this guide, we show you how to use **builders** to specify
+**filters** for your queries.
+
+Builders are classes provided by the MongoDB Java driver that allow you
+to use the builder pattern to interact with your MongoDB instance. The
+builders provided by the MongoDB Java driver can improve the readability
+of your code.  See the :doc:`Builders overview page
+</fundamentals/builders/>` for a list of Builders available in the Java
+driver. 
+
+
+Filters are the operations MongoDB uses to limit your results to what
+you want to see. Some examples of results from queries with filters are: 
+
+- Items that cost $0 to $25
+- A hotel with amenities that include an indoor swimming pool and free parking
+- A food critic review that mentions "spicy" 
+
+This guide shows you how to use builders with examples of the following
+types of operators: 
+
+- :ref:`Comparison <comparison>`
+- :ref:`Logical <logical>`
+- :ref:`Arrays <arrays>`
+- :ref:`Elements <elements>`
+- :ref:`Evaluation <evaluation>`
+- :ref:`Bitwise <bitwise>`
+- :ref:`Geospatial <geospatial>`
+
+The :java-core-api:`Filters <com/mongodb/client/model/Filters.html>`
+class provides static factory methods for all the MongoDB query
+operators. Each method returns an instance of the :ref:`Bson <bson>`
+type, which you can pass to any method that expects a query filter. 
+
+.. tip::
+
+   For brevity, you may choose to import all methods of the
+   :java-core-api:`Filters <com/mongodb/client/model/Filters.html>`
+   class statically: 
+
+   .. code-block:: java
+
+      import static com.mongodb.client.model.Filters.*;
+
+   The examples below assume this static import.
+
+The following sections show you how to use the ``Filters`` class to sort
+your queries using the following setup:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin imports
+   :end-before: end imports 
```

Comment:
I'm not sure what this section is supposed to show. A number of these imports aren't related to the filters. And we'd only want to show which imports we're using for a specific filter which you've covered by mentioning that they use the static import in the tip.